apprise/cli.py

```diff
@@ -241,16 +243,58 @@ def main(body, title, config, attach, urls, notification_type, theme, tag,
     # Create our Apprise object
     a = Apprise(asset=asset, debug=debug, location=ContentLocation.LOCAL)
 
-    # Load our configuration if no URLs or specified configuration was
-    # identified on the command line
-    a.add(AppriseConfig(
-        paths=[f for f in DEFAULT_SEARCH_PATHS if isfile(expanduser(f))]
-        if not (config or urls) else config,
-        asset=asset, recursion=recursion_depth))
-
-    # Load our inventory up
-    for url in urls:
-        a.add(url)
+    # The priorities of what is accepted are parsed in order below:
+    #    1. URLs by command line
+    #    2. Configuration by command line
+    #    3. URLs by environment variable: APPRISE_URLS
+    #    4. Configuration by environment variable: APPRISE_CONFIG
+    #    5. Default Configuration File(s) (if found)
+    #
```

Comment:
I guess i go a bit over-board with inline documentation. If i were to change this to a `"""`, would the string get parsed and assigned to memory (even though it's not being placed into any object) when generating the `.pyc` file?  With the commented code, the generated byte-file will just ignore the content. I do respect that [the docstring convention is used at the head of functions though](https://www.python.org/dev/peps/pep-0257/).  I didn't realize it can be used for inline too?

Reply:
Python is not my main language. But if I had to make an educated guess, I would assume that - after 20 years - CPython is smart enough to ignore unused inline strings for pyc compilation 😉